src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -1049,6 +1034,21 @@ public override void Accept(OperationVisitor visitor)
         }
     }
 
+    internal sealed partial class BoundDeconstructionVariables : BoundExpression
+    {
+        protected override OperationKind ExpressionKind => OperationKind.None;
+
+        public override void Accept(OperationVisitor visitor)
+        {
+            visitor.VisitNoneOperation(this);
```

Comment:
This is a dummy implementation, like `BoundTupleExpression`.
My understanding is that this implement IOperation, which John Hamby would take care of.